File: langs/Ruby/FizzBuzz.rb
x = gets.to_i

for i in 1..x do
  if i % 15 == 0
    print "FizzBuzz"
  elsif i % 5 == 0
    print "Buzz"
  elsif i % 3 == 0
    print "Fizz"
  else
    print i
  end
  print "/t"
end

File: langs/C++/FizzBuzz.cpp
#include <iostream>
using namespace std;

int main()
{
  int x;
  cin >> x;

  for (int i = 1; i <= x; i++)
  {
    if (i % 15 == 0)
    {
      cout << "FizzBuzz";
    }
    else if (i % 3 == 0)
    {
      cout << "Fizz";
    }
    else if (i % 5 == 0)
    {
      cout << "Buzz";
    }
    else
    {
      cout << i;
    }

    cout << "\t";
  }
}
